coffee-chats/src/main/webapp/src/components/NavBar.tsx

```diff
@@ -3,12 +3,17 @@ import React from "react";
 import {AppBar, Grid, Icon, IconButton, List, SwipeableDrawer, Toolbar, Tooltip} from "@material-ui/core";
 import {ListItemLink} from "./ListItemLink";
 
-function NavBarButtons(props: {onDrawerOpen: () => void}) {
+interface NavBarButtonsProps {
+  onDrawerOpen: () => void;
+  logoutUrl: string;
+}
+
+function NavBarButtons({onDrawerOpen, logoutUrl}: NavBarButtonsProps) {
   return (
       <Grid justify="space-between" container>
         <Grid item>
           {/* Left side of the navbar */}
-          <IconButton aria-label="Toggle navbar" onClick={props.onDrawerOpen}>
+          <IconButton aria-label="Toggle navbar" onClick={onDrawerOpen}>
```

Comment:
What's diff between using props and not?

Reply:
It's just because I added a separate type for props and moved the deconstruction to argument list